openmetadata-ui/src/main/resources/ui/src/components/NavBar/NavBar.tsx

```diff
@@ -74,14 +74,13 @@ import BrandImage from '../common/BrandImage/BrandImage';
 import CmdKIcon from '../common/CmdKIcon/CmdKIcon.component';
 import { useDomainProvider } from '../Domain/DomainProvider/DomainProvider';
 import { useGlobalSearchProvider } from '../GlobalSearchProvider/GlobalSearchProvider';
-import GithubStarModal from '../Modals/GithubStarModal/GithubStarModal.component';
-import WhatsNewAlert from '../Modals/WhatsNewModal/WhatsNewAlert/WhatsNewAlert.component';
 import WhatsNewModal from '../Modals/WhatsNewModal/WhatsNewModal';
 import NotificationBox from '../NotificationBox/NotificationBox.component';
 import { UserProfileIcon } from '../Users/UserProfileIcon/UserProfileIcon.component';
 import { useWebSocketConnector } from '../WebSocketProvider/WebSocketProvider';
 import './nav-bar.less';
 import { NavBarProps } from './NavBar.interface';
+import PopupAlertsCardsClassBase from './PopupAlertClassBase';
```

Comment:
```suggestion
import popupAlertsCardsClassBase from './PopupAlertClassBase';
```
To avoid confusion between class or object instance